assets/javascripts/modules/checkout/voucherFields.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import moment from 'moment'
+import formElements from './formElements'
+import CustomDatePicker from '../react/customDatePicker'
+import CharacterCountedTextArea from './characterCountedTextArea'
+import reviewDetails from './reviewDetails'
+
+require('react-datepicker/dist/react-datepicker.css');
+
+const MAX_WEEKS_AVAILABLE = 4;
+
+function filterDate() {
+    return (date) => date.day() === 1;
```

Comment:
Yeah, that might be more true to what you are actually getting, e.g. you'll see the first day you can actually use your voucher to get the paper. I'll keep it as it is and create a new PR for that specific change once this one is merged.